Objects/frameobject.c

```diff
@@ -306,7 +308,10 @@ framelocalsproxy_visit(PyObject *self, visitproc visit, void *arg)
 static PyObject *
 framelocalsproxy_iter(PyObject *self)
 {
-    return PyObject_GetIter(framelocalsproxy_keys(self, NULL));
+    PyObject* keys = framelocalsproxy_keys(self, NULL);
```

Comment:
Probably should error-check this too, and the PyList_Append in that function could fail so should clean up and return NULL.

Reply:
Also, there is a lack of error checking in some new "frameobject.c" functions. I will create a separate issue for this.

Reply:
I fixed the `framelocalsproxy_keys` related checks. We can address the others in a seperate PR?